AVInput/AVInput.cpp

```diff
@@ -1172,6 +1176,55 @@ std::string AVInput::getSPD(int iPort)
     return spdbase64;
 }
 
+uint32_t AVInput::setAudioMixerLevels(const JsonObject& parameters, JsonObject& response)
+{
+    returnIfStringParamNotFound(parameters, "primaryVolume");
```

Comment:
Please check the architecture for API name correction: https://www.stb.bskyb.com/confluence/pages/viewpage.action?spaceKey=2016&title=HDMI+Audio+Mixing